### DISCUSSION TITLE
How to resolve this issue...??? Pls help I am stucked by this issue from 2 days . can't figure out how to resolve....

### DISCUSSION THREAD

**Discussion by SjxSubham** [community/community#142418] [Programming Help]
### Body

![Screenshot 2024-10-23 213222](https://github.com/user-attachments/assets/e706a365-e4c8-4e13-b0ed-739917e49c65)

![Screenshot 2024-10-23 213407](https://github.com/user-attachments/assets/9d0a663d-ba71-45fd-941e-5053b8837ab4)
stuck into these git problem from 2 days
do u know how to fix this ???
whenever I'm trying to clone a repo this error occured (even whenever i want to push new changes)... other git commands are working...

steps I have already done- 
1. reinstalled git with its latest version. (although it's not a solution)
2. my internet works fine
3. It's a sudden error that I never faced this type of ssl issue before
4. I'hv also run this command ```  git config --global http.sslBackend openssl ``` but the problem isn't resolved. 
5. after running no 4. command I am getting this error ``` fatal: unable to access 'https://github.com/hka****19/terra***-learn.git/': Failure when receiving data from the peer ```

Help somebody if u have also facing this issue...
Is it any github operation server brokerage??
Any SSL bug???
 # cause I am facing this problem suddenly from 2 days, before it worked fine....

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MasterInQuestion** [comment]
"error:0A0001BB:SSL routines::bad record type"
    Potential network interference: if you can rule out the possibility of the software.
    Rough search indicated no such problem in regular build.

    Try:
    [curl](https://github.com/MasterInQuestion/ShellUtils/commit/34910eb5ca7ff55f02d337f46aa75c031277ec66) -v --max-filesize 1 "https://github.com/"
    ; to test.

    More on OpenSSL "bad record type":
    https://mailman.nginx.org/pipermail/nginx-devel/2022-September/5MHGU4ZHG2WSUUXPBZVJ5O36FOUONZMN.html
    https://forum.nginx.org/read.php?29,294928,294935

**Comment by hewageuhcu** [comment]
Did you try to get a new clone? Sometimes this happen as you do some changes in your GitHub repo after cloning it like creating a new branch. Then you have to remove the previous clone and get a new clone from your repo.

**Reply by SjxSubham** [reply]
Yes brother even i tried to clone a repo from your repos also...
![Screenshot 2024-10-24 184149](https://github.com/user-attachments/assets/bcb6e9c1-4a4c-41dd-9927-476d42f6b8eb)

**Reply by MasterInQuestion** [reply]
Hint:
    Use descriptive text instead of image may greatly improve the post's hit rate.
    Avoid mostly duplicate images. (this one much identical to main post one)
    And the title is undescriptive; consider change it to:
    OpenSSL "bad record type"
